Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/VAVSingleZoneTSupSet.mo

```diff
@@ -500,6 +498,16 @@ economizer dampers.
 Note that the inputs <code>uHea</code> and <code>uCoo</code> must be computed
 based on the same temperature sensors and control loops
 </p>
+<p>
+fixme, mg notes: 
```

Comment:
These notes are some details from the guideline, so I'm simply going to copy them over to my stack of detail related questions. They do not impact the general functionality of the sequence, in the large majority of the cases.